xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1137,7 +1137,7 @@ bool CPVRClients::UpdateAndInitialiseClients(bool bInitialiseAllClients /* = fal
       }
 
       if (bDisabled && (g_PVRManager.IsStarted() || g_PVRManager.IsInitialising()))
-        CGUIDialogOK::ShowAndGetInput(24070, 24071, 16029, 0);
+        CGUIDialogOK::ShowAndGetInput(24070, 16029);
```

Comment:
Is it by intention that you dropped the 24071 ("An unknown error has occurred.") part? IMO it's fine to be dropped, just want to be sure it was also your intention and not by accident.